charms/jupyter-controller/examples/sample-notebook.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kubeflow.org/v1
+kind: Notebook
+metadata:
+  annotations:
+  labels:
+    access-ml-pipleine: "true"
+    app: sample-notebook
+  name: sample-notebook
+spec:
+  template:
+    spec:
+      containers:
+      - name: notebook
+        image: kubeflownotebookswg/jupyter-tensorflow-full:v1.7.0
```

Comment:
Would be interesting to test the ROCK I published here ;-)

Reply:
Updated test.